custom_metrics/pwa.js

```diff
@@ -80,13 +80,18 @@ function getInfoForPattern(regexPattern, extractMatchingGroupOnly) {
   });
 }
 
+// Unlike serviceWorkerStrictRegistrationPattern that only matches SW registration scripts that contain URLs,
+// serviceWorkerLaxRegistrationPattern matches any call to the SW registration script (e.g. passing a variable, etc).
+const serviceWorkerLaxRegistrationPattern = /navigator\.serviceWorker\.register\(([^)]*)\)/g;
+const serviceWorkerInfo = getInfoForPattern(serviceWorkerLaxRegistrationPattern, true);
```

Comment:
I changed it to `serviceWorkerRegistration`, since serviceWorkerRegistration  might be confused with the [ServiceWorkerRegistration interface](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration).